### DISCUSSION TITLE
Nightscoutuploader error: Unable to do REST API download

### DISCUSSION THREAD

**Discussion by Pipperoo169** [NightscoutFoundation/xDrip#3170] [Bug reports]
Hi,
I have recently got a problem with uploading data to Nightscout (no data in Nightscout browser) and to Followers as a consequence.
Xiaomi Redme 9C NFC
I set up a new Libre 2 last night, this is the only step I have recently done.
Can you advise how this issue can be resolved, please?
![IMG_9909](https://github.com/NightscoutFoundation/xDrip/assets/123673646/7cb3e746-b183-44b9-b4bb-81991c358bc2)
![IMG_9910](https://github.com/NightscoutFoundation/xDrip/assets/123673646/ac596100-7ca9-4a37-9c3d-666a1d9c3a8f)

**Comment by Navid200** [comment]
@psonnera

**Reply by psonnera** [reply]
Does your Nightscout site ooen in a browser?

**Comment by Pipperoo169** [comment]
It opened in a browser but there was no data. But I found out what the problem was - issue with a mobile internet on the phone. Now it's fixed. Sorry for disturbing :)
